ui/src/app/components/cases/test-case-details.component.html

```diff
@@ -40,7 +40,7 @@
             *ngIf="!testCase?.deleted && false"
             [versionId]="testCase?.workspaceVersionId"
             [testCase]="testCase"></app-test-case-detail-pagination>
-          <div *ngIf="version?.workspace?.isWeb && !testCase?.deleted">
+          <div *ngIf="(version?.workspace?.isWeb || version.workspace.isMobileWeb) && !testCase?.deleted">
             <app-chrome-record-button></app-chrome-record-button>
```

Comment:
@nixonakash01 Have you tested if it is working for android and IOS as well ? Paste a screenshot of the app details for both android and IOS executions ?

Reply:
yes its working, ill add the screenshots
![image](https://user-images.githubusercontent.com/106724526/224619774-1b8f0cef-f226-4d32-8faf-ee8bd72aef3d.png)
![image](https://user-images.githubusercontent.com/106724526/224620137-98bc01be-1ce1-4ed5-b015-ef1ed95acef3.png)